COMET.Web.Common/Services/SessionManagement/SessionService.cs

```diff
@@ -609,6 +225,46 @@ public Task<Result> ReadEngineeringModels(IEnumerable<EngineeringModelSetup> eng
             return this.ReadEngineeringModels(engineeringModelSetups.Select(x => x.EngineeringModelIid));
         }
 
+        /// <summary>
+        /// Writes an <see cref="OperationContainer" /> to the <see cref="ISession" />
+        /// </summary>
+        /// <param name="operationContainer">The <see cref="OperationContainer" /> to write</param>
+        /// <param name="files">A <see cref="IReadOnlyCollection{T}"/> of the file paths as <see cref="string"/> to create or update</param>
+        /// <returns>A <see cref="Task{T}" /> with the <see cref="Result" /> of the operation</returns>
+        public async Task<Result> WriteTransaction(OperationContainer operationContainer, IReadOnlyCollection<string> files)

```

Comment:
would be nice to move that one in the SDK. Not for now but please open a Ticket on the SDK for it

Reply:
Done

https://github.com/RHEAGROUP/COMET-SDK-Community-Edition/issues/332